src/modules/collection/sagas.ts

```diff
@@ -660,27 +660,30 @@ export function* collectionSaga(legacyBuilderClient: BuilderAPI, client: Builder
         })
       ) // TODO: try to convert this to a generator so we can test it's called with the right parameters
       const allItemPages: PaginatedResource<Item>[] = yield queue.addAll(promisesOfPagesToFetch)
-      const itemsToApprove = allItemPages.flatMap(result => result.results)
+      const itemsWithPendingCurations = allItemPages.flatMap(result => result.results)
 
-      if (!itemsToApprove.length) {
+      if (!itemsWithPendingCurations.length) {
         throw Error('Error fetching items to approve')
       }
-      const { cheque, content_hashes: contentHashes, chequeWasConsumed }: ItemApprovalData = yield call(
+      const { cheque, content_hashes: contentHashes, chequeWasConsumed, root }: ItemApprovalData = yield call(
         [legacyBuilderClient, 'fetchApprovalData'],
         collection.id
       )
 
       // 3. Compute the merkle tree root & create slot to consume
       const tree = generateTree(Object.values(contentHashes))
-      const amountOfItemsToPublish = itemsToApprove.filter(item => !item.isApproved).length
+      const amountOfItemsToPublish = itemsWithPendingCurations.filter(item => !item.isApproved).length
 
       if (cheque.qty < amountOfItemsToPublish) {
         throw Error('Invalid qty of items to approve in the cheque')
       }
 
+      console.log('Root value', root)
+      console.log('Tree root', tree.merkleRoot)
```

Comment:
👀

Reply:
I removed the logs just to put them again in another commit LOL